src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card{
+  display: block;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  // #region card info
+  &__image {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__text {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+   }
+
+  &__code {
+    color: #{$secondary};
+    margin-bottom: 16px;
+    font-weight: 400;
+  }
+
+  // #endregion card info
+  &__stars{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-text {
+      align-items: flex-end;
+    }
+
+  }
+
+  // #region price
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: bold;
+      line-height: 18px;
+    }
+  }
+
+  // #endregion price
+
+  &__button {
+    display: flex;
+    width: 166px;
+    height: 40px;
+    align-items: center;
+    justify-content: center;
+    background-color: #00ACDC;
+    border-radius: 5px;
+
+    color: #FFF;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      background-color: #FFF;
+      border: 1px solid  #00ACDC;
+      cursor: pointer;
+      color: #00ACDC;
+    }
+ }
+
+ &:hover {
+  transform: scale(1.20);
+  transition: transform 0.3s;
```

Comment:
This hover effect should be smooth when mouse enter and when it leaves, but at the moment transition work only when mouse enter on card block